src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/OrchardCore.Data.YesSql.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <RootNamespace>OrchardCore.Data</RootNamespace>
+    <RootNamespace>OrchardCore.Data.YesSql</RootNamespace>
```

Comment:
Not sure about this one

Reply:
I thought that someone might comment on this one :) but while the `OC.Data.YesSql` has its own project it would be better to use YesSql at the end of the namespace, but you still can use `OC.Data` namespace

Again if there's a type on `OC.Data.YesSql` project and `OC.Data` namespace, it might let us ask why it's not on `OC.Data` especially if it's not related to YesSQL